ibm_catalog.json

```diff
@@ -0,0 +1,159 @@
+{
+  "products": [
+    {
+      "label": "Cloud automation for Cloud Monitoring Agent",
+      "name": "deploy-arch-ibm-monitoring-agent",
+      "product_kind": "solution",
+      "tags": [
+        "ibm_created",
+        "logging",
+        "logging_agent",
```

Comment:
`logging` and `logging_agent` are not valid tags. Use `logging_monitoring`.

for list of valid tags run `ibmcloud catalog filter options --all`